conf/rcac.config

```diff
@@ -0,0 +1,31 @@
+//Profile config names for nf-core/configs
+params {
+        config_profile_description = 'The Rosen Center for Advanced Computing (RCAC) at Purdue University cluster profile provided by nf-core/configs.'
+        config_profile_contact = 'Yucheng Zhang(zhan4429@purdue.edu)'
+        config_profile_url = 'https://www.rcac.purdue.edu'
+}
+
+params {
+        max_memory = 1024.GB
+        max_cpus = 128
+        max_time = 336.h
+}
+
+process {
+        executor = 'slurm'
+        clusterOptions = '-N 1 -n 1'
+ }
+
+executor {
+        queueSize = 50
+        pollInterval = '1 min'
+        queueStatInterval = '5 min'
+        submitRateLimit = '10 sec'
+}
+
+// Set $NXF_SINGULARITY_CACHEDIR in your ~/.bashrc
+// to stop downloading the same image for every run
```

Comment:
It would be better to describe this in `docs.md`  - users are not exposed to this specific file :)